examples/quick-start-datadog-nodejs/test-api.yaml

```diff
@@ -2,7 +2,7 @@ type: Test
 spec:
   id: W656Q0c4g
   name: http://app:8080
-  description: akadlkasjdf
+  description: Calling Hello World API
```

Comment:
What was the problem with `akadlkasjdf`?

Reply:
My documentation OCD won't free me if if don't update this description 😂